pkg/config/config.go

```diff
@@ -390,11 +390,19 @@ type Config struct {
 	// Public: No
 	IsForwardOnly bool `yaml:"is_forward_only" envconfig:"is_forward_only" public:"false"`
 
-	// IsSecureForwardOnly has the same behaviour as `IsForwardOnly` but with some inventory data and a heartbeat
+	// IsSecureForwardOnly has the same behaviour as the default but without sending host metrics.
+	// It creates the host entity, sends inventory data and does a heartbeat to not expire the host entity,
+	// it also sends integrations data.
 	// Default: False
 	// Public: No
 	IsSecureForwardOnly bool `yaml:"is_secure_forward_only" envconfig:"is_secure_forward_only" public:"false"`
 
+	// IntegrationsOnly has the same behaviour as the default but without creating the host entity on entity platform and
+	// without sending host metrics, it notifies EP to skip the entity by sending the integrations_only inventory entry.
+	// Default: False
+	// Public: No
+	IntegrationsOnly bool `yaml:"integrations_only" envconfig:"integrations_only" public:"false"`
```

Comment:
Shall we add a condition to ensure that these modes are exclusive? Maybe not for the current ones as we could break existing confs, but we could check that if `integrations_only` is enabled the other modes are disabled. wdyt?

Reply:
Agreed, I will add it